src/es.json

```diff
@@ -1110,8 +1110,8 @@
             "taskType": "Tipo de tarea",
```

Comment:
Use this.
"timelineCell": "Celda de la línea de tiempo",

Reply:
Changed